### PR TITLE
README: update communication section

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
README: update communication section

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @gundalow i used a template from https://github.com/ansible-community/project-template#communication we recently created.
But i wonder if we should go further and even remove the Real-time channels section.. thoughts?

File: README.md [hunk 1]
```diff
@@ -29,22 +29,27 @@ If you're interested in becoming a maintainer of this collection, refer to the [
 
 ## Communication
 
-We announce important development changes and releases through Ansible's [The Bullhorn newsletter](https://github.com/ansible/community/wiki/News#the-bullhorn). If you are a contributor, be sure you are [subscribed](https://eepurl.com/gZmiEP).
+> `GitHub Discussions` feature is disabled in this repository. Use the `network` tag on the forum in the [Project Discussions](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=network) or [Get Help](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=help&tags=network) category instead.
```
Comment (gundalow) [inline README.md:32]: Why even mention GitHub discussions if it's disabled?

File: README.md [hunk 2]
```diff
@@ -29,22 +29,27 @@ If you're interested in becoming a maintainer of this collection, refer to the [
...
 
-Also join us on [Slack](https://ansiblenetwork.slack.com).
+### Real-time channels
 
-Contributors to this collection take part in the global [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track [The Bullhorn newsletter](https://eepurl.com/gZmiEP) and join us.
+* Matrix rooms:
+    * [#network:ansible.com](https://matrix.to/#/#network:ansible.com): dedicated to network automation.
+    * [#users:ansible.com](https://matrix.to/#/#users:ansible.com): general use questions and support.
+    * [#social:ansible.com](https://matrix.to/#/#social:ansible.com): say hello or share a funny joke and let's laugh together;)
+    * [#ansible-community:ansible.com](https://matrix.to/#/#community:ansible.com): community and collection development questions.
```
Comment (gundalow) [inline README.md:50]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (samccann) [reply README.md:32]: I think it's better to use just the network tag link, not the new topic link. SEO will pick up common links and raise visibility of that content on search results.

**Comment by samccann** [comment]
My understanding is yes, we should ONLY list the forum at this point. If the technology has a forum group, link to that. If not, link to an appropriate tag, as you've done here.

Based on the results of this discussion, can you update [the hackmd with some general guidelines ](https://hackmd.io/nbbL7VwAR2u8OXVdSNpGcQ) pls? 

And if you are looking for a github issue to link to, it's https://github.com/ansible-community/community-team/issues/554

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md]: I don't remember the original rationale, took if from another repo. Yes, let's remove it.

**Comment by gundalow** [comment]
Can we update `CONTRIBUTING.md` at the same time to point to the new docs

**Comment by Andersson007** [comment]
> Can we update `CONTRIBUTING.md` at the same time to point to the new docs

@gundalow SGTM, updated, PTAL

**Review by oraNod — APPROVED** [review]
This seems fine to me. I had a quick scan through the repo and didn't see any other spots to update. Thanks @Andersson007

**Review by samccann — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Contributing
 
-Refer to the [Ansible Contributing guidelines](https://github.com/ansible/community-docs/blob/main/contributing.rst) to learn how to contribute to this collection.
+Refer to the [Ansible Contributing guidelines](https://docs.ansible.com/ansible/devel/community/index.html) to learn how to contribute to this collection.
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply CONTRIBUTING.md]: Makes sense, thanks for spotting!

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

**Comment by mariolenz** [comment]
4 approvals... time to merge this?

**Comment by Andersson007** [comment]
thanks everyone!

**Pull request merged**